documentation/modules/exploit/unix/webapp/trixbox_ce_auth_rce.md

```diff
@@ -43,7 +43,7 @@ Exploit target:
 
    Id  Name
    --  ----
-   0   Trixbox CE <= v2.8.0.4
+   0   Trixbox CE v2.8.0.4
```

Comment:
If this affects older versions and you have confirmed this, please revert this change. Only the title of the exploit module needed to be changed.

Reply:
I think this is the same file??? I can't see why I commented on this twice. Confirmed resolved.